Makefile

```diff
@@ -65,6 +65,7 @@ guess_platform_script := \
 guess_platform := $(shell $(guess_platform_script))
 
 all: all-$(guess_platform)
+check: check-$(guess_platform)
 
```

Comment:
Good idea.  Would it be better to add these to the check-% rules so that they can be run for a specific platform?